_posts/en/2021-06-08-working-in-the-open-part-2.md

```diff
@@ -0,0 +1,173 @@
+Or, you may feel less receptive to feedback as you get closer to a final version of a document since you spent so much time aligning your ideas together and wordsmithing.
+
+As explained in our previous post, working in the open is a participatory approach which includes public reflection and documentation.
+The process itself is as important as the final document.
+
+### Be Humble and Open Minded
+
+Working in the open may seem like putting yourself out there, subject to criticism and judgement of others.
+
+But remember that no one knows everything.
+Opening up is an opportunity for you and your team to share what you are working on and enable unique collaboration experiences with colleagues and citizens alike.
+
+This approach is what made open source software projects so powerful and adopted across the whole world.
+It is also a good way to align with the Directive on Open Government as mentioned earlier.
+
+Of course, the official review process can still occur for each major revision of your documents.
+You simply ensure that in between such versions, the creation process is done openly and enables folks from all backgrounds to contribute.
+
+### Manage the Community
+
+An argument sometimes used for avoiding working in the open is that managing the community may end up being more work than what you get back.
+Or that receiving external contributions before a certain version is mature enough would slow you down.
+
+Based on our experience, small and continuous feedback makes the process much less cognitively intensive since contributions usually are smaller and can be more easily integrated in the workflow.
+It also helps identify issues early on so that side discussions may occur on those specifically.
+And by continuously reviewing and integrating the feedback, it helps minimize the disruption of the entire creation effort.
+
+As mentioned before, this approach has to be participatory.
+It means that you have to set the stage, explain what you are trying to accomplish and welcome those contributions from all around.
+
+And encourage external folks to comment.
+Make sure you reply to them either by simply acknowledging you have read their contributions or by letting them know what will be done about it.
+
+Since we abide by a code of values and ethics, clearly outline what is tolerated and what is not.
+Be clear within your team that this is something that is beneficial and help new comers to feel comfortable with the process.
+
+You may be surprised as to who ends up contributing!
+Be welcoming, open minded, and inclusive.
+Not everyone has the same background and life experiences.
+We can learn from everyone and grow better together!
+
+#### Further Readings
+
+This post is getting a bit long but working in the open is something we should all strive to get accustomed with.
```

Comment:
The post is not too long.  It reads well and its engaging throughout.  I would remove that reference.

Reply:
Will do! :+1: